packages/jbrowse-plugin-apollo/src/components/ImportFeatures.tsx

```diff
@@ -46,6 +54,64 @@ export function ImportFeatures({
     )
   }
 
+  function handleDeleteFeatures(e: React.ChangeEvent<HTMLInputElement>) {
+    setDeleteFeatures(e.target.checked)
+  }
+
+  // fetch and set features count for selected assembly
+  useEffect(() => {
+    if (assemblyId) {
+      updateFeaturesCount()
```

Comment:
The entire `updateFeaturesCount` function should be inside this `useEffect` block. It's an idiomatic React thing.